lib/AddDependencies.js

```diff
@@ -122,7 +125,7 @@ class AddDependencies {
   }
 
   saveToPackage() {
-    const packageFilePath = 'package.json';
+    const packageFilePath = `${this.packageDir}package.json`;
```

Comment:
According to line 44, `this.packageDir` for an argument like `/home/user/project/package.json` will be `/home/user/project/package.json`. Thus, here we'll get `/home/user/project/package.jsonpackage.json`

Reply:
Opps, fixing

Reply:
I suggest to use `this.packageFile` instead, and its initial value will be `./package.json`.

Reply:
already done 👍